README.md

```diff
@@ -85,6 +85,12 @@ An alternative to the Grunt build is provided via the `util.sh` shell script.
 1.  `mkdir google_closure_templates; cd google_closure_templates`
 1.  `curl -O https://dl.google.com/closure-templates/closure-templates-for-javascript-latest.zip`
 1.  `unzip closure-templates-for-javascript-latest.zip`
+1.  `git clone https://github.com/google/closure-templates.git`
```

Comment:
Do you know if this will actually build all the Javascript bits too?  If so, we can probably delete the previous steps about downloading from dl.google.com (and add a line or two about copying the SoyToJsSrcJar to the directory as well).  That would actually be preferable, as I don't think that release is necessarily the most up-to-date.